win32.js

```diff
@@ -29,23 +29,23 @@ module.exports = {
         }
       ]
 
-      let win32metadata = Object.assign({}, opts['version-string'], opts.win32metadata)
+      let win32metadata = Object.assign({}, opts.versionString, opts.win32metadata)
 
       var rcOpts = {'version-string': win32metadata}
 
-      if (opts['app-version']) {
-        rcOpts['product-version'] = rcOpts['file-version'] = opts['app-version']
+      if (opts.appVersion) {
+        rcOpts['product-version'] = rcOpts['file-version'] = opts.appVersion
       }
 
-      if (opts['build-version']) {
-        rcOpts['file-version'] = opts['build-version']
+      if (opts.buildVersion) {
+        rcOpts['file-version'] = opts.buildVersion
       }
 
-      if (opts['app-copyright']) {
-        rcOpts['version-string'].LegalCopyright = opts['app-copyright']
+      if (opts.appCopyright) {
+        rcOpts.versionString.LegalCopyright = opts.appCopyright
```

Comment:
This is incorrect, `rcOpts` is for the `rcedit` module. I take it you did a global search/replace?